functions/nodejs-apps.bash

```diff
@@ -9,7 +9,7 @@ nodejs_setup() {
   if node_is_installed && ! is_armv6l; then return 0; fi
```

Comment:
@mstormi 
Can you explain why `&& ! is_armv6l` is required for the early return?

Reply:
no

Reply:
Okay.

The only reason I can think of, is to update NodeJS on `armv6l` systems, because we have package repo for thosee and therefore no package manager to update.